omod/pom.xml

```diff
@@ -255,6 +264,20 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Patient search criteria module -->
+        <dependency>
+		    <groupId>com.github.Reyano132.openmrs-module-patientsearchcriteria</groupId>
```

Comment:
please revise this groupid to match our conventions https://wiki.openmrs.org/display/docs/Module+Conventions

Reply:
@kaweesi sir, here I used jitpack to host my remote repository.  Jitpack added "com.github.Reyano132" prefix to the group ID. Once we officially host the patient search criteria module (https://github.com/openmrs/openmrs-module-patientsearchcriteria), I'll change the groupId value here.